15-microbiome-community.Rmd

```diff
@@ -15,10 +15,27 @@ data("GlobalPatterns")
 tse <- GlobalPatterns
 ```
 
-
 ## Community composition
 
-Heatmaps
+Community composition can be visualized with heatmap where other axis represent
+samples and another taxa. Color of each intersection point represent abundance
+of a taxon in a specific sample. 
+
+Here, clr (center log ratio) transformed abundances are plotted at Phylum level. 
+
+```{r heatmap}
+library(ggplot2)
+tse_phylum <- agglomerateByRank(tse, rank = "Phylum")
+tse_phylum <- transformSamples(tse_phylum, method = "clr", pseudocount = 1)
```

Comment:
I propose adding here transformFeatures(..., method = "Z") as this supports the visualization and is relatively common.

Reply:
clr --> z

Reply:
clr (on samples) + z (on features)